lib/core/Executor.js

```diff
@@ -1,9 +1,10 @@
 'use strict';
 
-const PROVISION = 'provision';
-const REVOKE    = 'revoke';
-const SHOW      = 'show';
-const VERSION   = '0.0.1';
+const PROVISION   = 'provision';
+const REVOKE      = 'revoke';
+const SHOW        = 'show';
+const FETCH_BATCH = 'fetchBatch';
+const VERSION     = '0.0.1';
```

Comment:
I think we need to bump this version.
But, this version is different from the one defined in `package.json` https://github.com/cermati/iamx/blob/master/package.json#L3
You might want to confirm with @sdsdkkk regarding the versioning.

Reply:
updated to `2.0,.0` for this VERSION and package version